discordrus_test.go

```diff
@@ -26,6 +26,7 @@ func init() {
 			Author:             "",                     // Setting this to a non-empty string adds the author text to the message header
 			DisableTimestamp:   false,                  // Setting this to true will disable timestamps from appearing in the footer
 			TimestampFormat:    "Jan 2 15:04:05.00000", // The timestamp takes this format; if it is unset, it will take logrus' default format
+			TimestampLocale:    time.UTC,               // The timestamp takes it's timezone from the provided locale
```

Comment:
Previously, TimestampLocale was not specified so the entry's time would default to `time.Local`, so we should use `time.Local` instead.

Reply:
✔